src/ctx.rs

```diff
@@ -62,6 +62,40 @@ impl Ctx {
             builder = builder.tls_connector(std::sync::Arc::new(tls_connector));
         }
 
+        #[cfg(feature = "rustls-tls")]
```

Comment:
This makes it so those who compile with native-tls can't add custom certs.

Reply:
Sorry but can you elaborate more ? I think it's already a seperate code block that's outside of `#[cfg(feature = "native-tls")]` so that those who compile with `native-tls` will correctly use the code block under `#[cfg(feature = "native-tls")]`.

Reply:
Sorry was reviewing on mobile and didn't recognize this was just entire duplication of the native-tls block.